src/components/AnalyticsOptInModal.vue

```diff
@@ -23,14 +59,14 @@
 <script>
 import Modal from '@/components/Modal'
 import { mapActions, mapState } from 'vuex'
-
 export default {
   components: {
     Modal
   },
   data: function () {
     return {
-      loading: false
+      loading: false,
+      analyticsAccepted: null
```

Comment:
In this case you can access to `state.analytics?.acceptedDate` and use `mappState` to access it as a computed property